app/routes/xiview.py

```
@@ -38,8 +38,9 @@ async def get_peaklist(id, sd_ref, upload_id):
     }
     return Response(orjson.dumps(unpacked_data), media_type='application/json')
 
+
 @xiview_data_router.get('/visualisations/{project_id}', tags=["xiVIEW"])
-def visualisations(project_id: str, request: Request, session: Session = Depends(get_session)):
+def visualisations(project_id: str, request: Request, session: Session = Depends(get_session)): #  request not used?
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)